### ISSUE TITLE
Installed .desktop file has Hidden=true setting

### ISSUE THREAD

**Issue by neildarlow** [flathub/org.audacityteam.Audacity#66]
Hi,

Running the flatpak creates a .local/share/applications/org.audacityteam.Audacity.desktop file with Hidden=true resulting in no visible menu entry.

I checked upstream's audacity.desktop.in and it contains no such setting so, I think, this is something created for the flatpak. Changing the setting to Hidden=false results in the Sound & Video menu showing an Audacity entry.

This was installed and tested on Linux Mint 20.1 Ulyssa Cinnamon edition.

Audacity 3.0.0 flatpak info:
    Audacity - Record and edit audio files

          ID: org.audacityteam.Audacity
         Ref: app/org.audacityteam.Audacity/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 3.0.0
     License: GPL-2.0 and CC-BY-3.0
      Origin: flathub
    Collection: org.flathub.Stable
    Installation: system
    Installed: 61.4 MB
     Runtime: org.gnome.Platform/x86_64/3.38
         Sdk: org.gnome.Sdk/x86_64/3.38

      Commit: 0a6fea322c6c0e63ab7bff30d5a0054a471d751673968e872922e6ffcbacb14d
      Parent: be192c261406e248ee7ecb70303307fc3fc74b10342d3b5cde29accb844cce55
     Subject: Allow network access for IPC (Fixes #64) (5f86e949)
        Date: 2021-03-29 14:59:26 +0000

**Comment by eszlari** [comment]
I cannot reproduce this under Kubuntu 20.10, the file `.local/share/applications/org.audacityteam.Audacity.desktop` is not created here.

**Comment by neildarlow** [comment]
Hi @eszlari,

This is interesting, I also have org.gnome.Platform/x86_64/40 installed which was updated yesterday.

I observed this issue multiple times (by completely uninstalling Audacity and removing its user environment) before the update. Since the update of that later runtime the Audacity flatpak is providing a launcher in the Sound & Video menu and there is no .desktop file installed in .local/share/applications as you described.

All very odd but working now, so I'll close this.

Thanks for your time.
